psycop/common/model_training_v2/config/historical_registry_configs/data/dataloader_filterer/dataloader_filterer_20231127_131457.cfg

```diff
@@ -0,0 +1,12 @@
+
+[dataloader_filterer]
```

Comment:
Perhaps we can come up with a sexier name here? It's blurring the lines between data-loading and preprocessing. I don't know what it should be off the top of my head, but we want to indicate how it is different from preprocessing steps.

Or perhaps it could just take preprocessing steps? I'm uncertain here.